tests/ImageSharp.Benchmarks/Codecs/Jpeg/DecodeJpeg_ImageSpecific.cs

```diff
@@ -49,18 +47,15 @@ public ShortClr()
         [Params(
             TestImages.Jpeg.BenchmarkSuite.Lake_Small444YCbCr,
             TestImages.Jpeg.BenchmarkSuite.BadRstProgressive518_Large444YCbCr,
+            /* The scaled result for the large image "ExifGetString750Transform_Huge420YCbCr"
```

Comment:
The `/*   */` are harder to work with in most C# IDE-s. Was the change necessary because of the rule? If yes, then I'm afraid that enforcing it may affect our test-code productivity. Or even worse: not placing comments where they should be present.

Reply:
I'd have added the spaces.

Reply:
There are two conflicting rules here:

- SA1515	Single-line comment should be preceded by blank line
- SA1115	The parameter should begin on the line after the previous parameter.

I have changed it back now as it was before with an exception for SA1115, see commit: d59ca988a0e730db1bfbac9825593e58f2b8b1e9